frontend/src/components/doctors/doctors/DoctorSmallCard.tsx

```diff
@@ -15,23 +22,50 @@ function DoctorSmallCard({
     distanceUnit = "mi",
     onClick,
 }: DoctorSmallCardProps) {
-    const distance =
+    const closestLocation =
         locationForDistanceCalculation &&
-        getDoctorMinimalDistance(doctor, locationForDistanceCalculation, distanceUnit);
+        getDoctorNearestLocation(doctor, locationForDistanceCalculation);
+
+    // TODO: replace with the real fields
+    const averageRating = 4.5;
+    const totalReviews = 10;
 
     return (
         <div className="doctorSmallCard" onClick={onClick}>
-            {doctor.fullName}
-            <br />
-            {doctor.category}
-            <br />
-            {doctor.specialities}
-            <br />
-            {distance && distance !== Infinity && (
-                <p>
-                    Distance: {distance.toFixed(2)} {distanceUnit}
-                </p>
-            )}
+            <DoctorImage doctor={doctor} />
+            <div className="doctorSmallCardData">
+                <div>
+                    <h5 className="text-nowrap">{doctor.fullName}</h5>
+                </div>
+                <div className="row w-100 m-0 flex-nowrap">
+                    <div className="doctorCategoryRibbon col-auto">
+                        <p className="text-black">{doctor.category || ""}</p>
+                    </div>
```

Comment:
Do we need a ribbon if there is no category?
In addition, I think it should be another separated component